layouts/contact/single.html

```diff
@@ -0,0 +1,34 @@
+{{ partial "general-title" . }}
+
+{{ partial "header" . }}
+    {{ partial "navbar" . }}
+    <!-- Main -->
+    <div id="main">
+        <article class="post">
+            {{ $.Scratch.Set "h1" true }}
+            {{ .Render "header" }}
+            <div id="content">
+                {{ .Content }}
+                <div class="contact-container">
+                <form id="contact" action="//formspree.io/{{ .Site.Social.email }}" method="post" center width="300px">
```

Comment:
That would make it more customizable and I can't imagine it would matter if it was .Site.Social.email or .Site.Social.formAction.